lib/util/cache/repository/types.ts

```diff
@@ -26,6 +26,12 @@ export interface BranchUpgradeCache {
   sourceUrl?: string;
 }
 
+export interface OnboardingBranchCache {
+  branchName: string;
```

Comment:
```suggestion
  onboardingBranch: string;
```

Reply:
Are you using it/ checking it later once it's set?

Reply:
The branchName? No.

Reply:
So should it be left out?

Reply:
Can be.